commit 4701d1699d34aa30d53c2c3cb502ca4f28562222
Author: Chanwoo Choi <cw00.choi@samsung.com>
Date:   Tue Jan 31 15:38:16 2017 +0900

    PM / devfreq: Fix available_governor sysfs
    
    commit bcf23c79c4e46130701370af4383b61a3cba755c upstream.
    
    The devfreq using passive governor is not able to change the governor.
    So, the user can not change the governor through 'available_governor' sysfs
    entry. Also, the devfreq which don't use the passive governor is not able to
    change to 'passive' governor on the fly.
    
    Fixes: 996133119f57 ("PM / devfreq: Add new passive governor")
    Signed-off-by: Chanwoo Choi <cw00.choi@samsung.com>
    Signed-off-by: MyungJoo Ham <myungjoo.ham@samsung.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1260_4701d1699d34aa30d53c2c3cb502ca4f28562222.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1260_4701d1699d34aa30d53c2c3cb502ca4f28562222.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
